dashboards/google-filestore/filestore_dashbard.json

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
The filename has a typo that needs to be fixed for it to correctly show up in the UI.

Reply:
The json should also exclude the "etag" and "name" fields.